tools/Plotting_data.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
I would rather not have python notebooks in mrcpp

Reply:
This one will be removed.